src/CounterInterceptor/Counts.cs

```diff
@@ -0,0 +1,54 @@
+public record Counts
```

Comment:
No benefit to making a record, might as well make a class. Also, make sure you put the namespace on this file.

Reply:
fixed